src/audio-worklet/free-queue/examples/hello-webassembly/sink-processor.js

```diff
@@ -0,0 +1,38 @@
+import FreeQueue from '../../src/free-queue.js';
+
+const RENDER_QUANTUM = 128
+
+/**
+ * A simple AudioWorkletProcessor node that will act as sink.
+ * It will pull out audio from queue and render it.
+ * 
+ * @class BasicProcessor
+ * @extends AudioWorkletProcessor
+ */
+class SinkProcessor extends AudioWorkletProcessor {
+  constructor(options) {
+    super();
+    // Initialize the Queue.
```

Comment:
Add a blank line at L15.

Reply:
Perhaps:
```
// Initializes a FreeQueue object with the option passed from the
// AudioWorkletNode constructor on the main thread.
```

I don't think we need to mention the structured cloning here.

Reply:
I thought of mentioning so because doing `Object.setPrototypeOf(this.queue, FreeQueue.prototype);` might seem weird to people at first